src/main/webapp/src/app/find-gardens-page/find-gardens.component.ts

```diff
@@ -8,9 +32,26 @@ import {OwlOptions} from 'ngx-owl-carousel-o';
     '../common/growpod-page-styles.css',
   ],
 })
+
+/**
+ * This is a component designed to display gardens close to either the current
+ * user's zipcode or a provided zipcode.
+ *
+ * This page takes an argument 'zip-code', the zipcode to search in.
+ * If the 'zip-code' argument is not provided, the component will call
+ * GET: /find-gardens. Otherwise, the component will call GET:
+ * /find-gardens?zip-code={zip-code}, with the value of the string 'zip-code' as {zip-code}.
+ */
 export class FindGardensComponent implements OnInit {
-  /* Option to display a certian number of slides based off of browser window size along with
-   ** some style options
+  displayInfo: Array<Garden> | null;
```

Comment:
nit: `gardenList` instead of `displayInfo`. It will make the HTML structure more readable.

Reply:
Will change.